File: data/src/main/java/com/github/gmetal/data/entity/ProductionCompanyEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class ProductionCompanyEntity {

    @JsonField
    public String name;
    @JsonField
    public Integer id;
}

File: data/src/main/java/com/github/gmetal/data/entity/ProductionCompanyEntity.kt
package com.github.gmetal.data.entity

import com.bluelinelabs.logansquare.annotation.JsonField
import com.bluelinelabs.logansquare.annotation.JsonObject

@JsonObject
class ProductionCompanyEntity {

    @JsonField
    var name: String? = null
    @JsonField
    var id: Int? = null
}
